R/quanteda_options.R

```diff
@@ -167,9 +167,10 @@ get_options_default <- function() {
                  base_featname = "feat",
                  base_compname = "comp",
                  language_stemmer = "english",
-                 pattern_hashtag = "#\\w+#?",
+                 pattern_hashtag = "#[\\p{L}\\p{N}]+#?",
```

Comment:
We actually had this wrong before, since according to Twitter, a purely numeric (included in \w or \p{Nd}) sequence following the `#` is not a valid hashtag. So, 
#420day OK
#420 not ok

Reply:
Ha, the irony, GitHub converted the second into a reference to an ancient issue about regexes.

Reply:
Actually we no longer need the patterns, because we have the same in our custom rules.
https://github.com/quanteda/quanteda/blob/33813d2e80c932bad2f690a7bba9cdaa42f369ae/rules/cusotm.yml#L18-L19
However, it is still nice to provide easy way to customize rules. We could make them `pattern_hashtag` and `pattern_hashtag` user name by default if you want. We could even rename it `tokens_tokenizer_rules` or something.